resources/authgear/templates/de/translation.json

```diff
@@ -17,61 +17,102 @@
   "error-verification-code-invalid": "Der Verifizierungscode ist ungültig, verwendet oder abgelaufen. Bitte fordern Sie einen neuen an.",
   "error-verification-code-invalid-click-to-resend": "Klicken Sie zum erneuten Senden.",
+  "v2.error.remove-last-identity": "Sie können die letzte Identität nicht trennen. Sie müssen mindestens 1 Identität behalten.",
   "error-remove-last-identity": "Sie können die letzte Identität nicht trennen. Sie müssen mindestens 1 Identität behalten.",
+  "v2.error.remove-last-primary-authenticator": "Sie können den letzten primären Authentifikator nicht entfernen. Sie müssen mindestens 1 primären Authentifikator für eine Identität behalten.",
   "error-remove-last-primary-authenticator": "Sie können den letzten primären Authentifikator nicht entfernen. Sie müssen mindestens 1 primären Authentifikator für eine Identität behalten.",
+  "v2.error.remove-last-secondary-authenticator": "Sie können den letzten sekundären Authentifikator nicht entfernen. Sie müssen mindestens 1 sekundären Authentifikator behalten, um die 2-Faktor-Authentifizierung aktiviert zu lassen.",
   "error-remove-last-secondary-authenticator": "Sie können den letzten sekundären Authentifikator nicht entfernen. Sie müssen mindestens 1 sekundären Authentifikator behalten, um die 2-Faktor-Authentifizierung aktiviert zu lassen.",
   "error-new-password-typo": "Das erneut eingegebene neue Passwort stimmt nicht mit dem neuen Passwort überein. Bitte stellen Sie sicher, dass beide übereinstimmen.",
   "error-web-ui-invalid-session-retry": "<p>Diese Seite ist ungültig oder abgelaufen.</p><p>Bitte kehren Sie zurück und wiederholen Sie Ihre Anfrage.</p>",
   "error-web-ui-invalid-session-return": "<p>Diese Seite ist ungültig oder abgelaufen.</p><p>Bitte kehren Sie zurück und wiederholen Sie Ihre Anfrage.</p>",
   "error-web-ui-invalid-session-action": "Zurück",
+  "v2.error.rate-limited": "Bitte warten Sie einen Moment, bevor Sie es erneut versuchen.",
   "error-rate-limited": "Bitte warten Sie einen Moment, bevor Sie es erneut versuchen.",
+  "v2.error.sms-send-limit-exceeded": "Sie haben das Limit für das Senden von Einmalpasswörtern überschritten. Bitte versuchen Sie es später erneut.",
   "error-sms-send-limit-exceeded": "Sie haben das Limit für das Senden von Einmalpasswörtern überschritten. Bitte versuchen Sie es später erneut.",
+  "v2.error.usage-limit-exceeded": "[An den Entwickler] Ihr Projekt hat das Nutzungslimit überschritten. Bitte überprüfen Sie Ihr Abonnement.",
   "error-usage-limit-exceeded": "[An den Entwickler] Ihr Projekt hat das Nutzungslimit überschritten. Bitte überprüfen Sie Ihr Abonnement.",
+  "v2.error.webhook-disallowed": "Der Vorgang ist nicht zulässig",
   "error-webhook-disallowed": "Der Vorgang ist nicht zulässig",
+  "v2.error.webhook-pre-signup-disallowed": "Die Anmeldung ist nicht zulässig",
   "error-webhook-pre-signup-disallowed": "Die Anmeldung ist nicht zulässig",
-  "error-webhook-disallowed-action": "Zurück",
```

Comment:
Your script is not supposed to remove anything.

Reply:
Yes if the key is unused originally. Or do you expect me do open another PR for that?

```
(authgear-main) $ rg 'error-webhook-disallowed-action' ./**/*.html
(authgear-main) $
```